profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.context.inc

```diff
@@ -0,0 +1,134 @@
+  $export = array();
+
+  $context = new stdClass();
+  $context->disabled = FALSE; /* Edit this to true to make a default context disabled initially */
+  $context->api_version = 3;
+  $context->name = 'newsroom';
+  $context->description = '';
+  $context->tag = 'newsroom';
+  $context->conditions = array(
+    'views' => array(
+      'values' => array(
+        'newsroom' => 'newsroom',
+        'newsroom-agenda' => 'newsroom-agenda',
+      ),
+    ),
+  );
+  $context->reactions = array(
+    'block' => array(
+      'blocks' => array(
+        'views-newsroom_navigation-type' => array(
+          'module' => 'views',
+          'delta' => 'newsroom_navigation-type',
+          'region' => 'sidebar_left',
+          'weight' => '-9',
+        ),
+        'views-39a3f985e3441350bd00477b466722e0' => array(
+          'module' => 'views',
+          'delta' => '39a3f985e3441350bd00477b466722e0',
+          'region' => 'sidebar_left',
+          'weight' => '-8',
+        ),
+      ),
+    ),
+  );
+  $context->condition_mode = 0;
+
+  // Translatables
+  // Included for use with string extractors like potx.
+  t('newsroom');
+  $export['newsroom'] = $context;
+
+  $context = new stdClass();
+  $context->disabled = FALSE; /* Edit this to true to make a default context disabled initially */
+  $context->api_version = 3;
+  $context->name = 'newsroom_item';
+  $context->description = '';
+  $context->tag = 'newsroom';
+  $context->conditions = array(
+    'node' => array(
+      'values' => array(
+        'newsroom_item' => 'newsroom_item',
+      ),
+      'options' => array(
+        'node_form' => '0',
+      ),
+    ),
+  );
+  $context->reactions = array(
+    'block' => array(
+      'blocks' => array(
+        'nexteuropa_newsroom-newsroom_related_content' => array(
+          'module' => 'nexteuropa_newsroom',
+          'delta' => 'newsroom_related_content',
+          'region' => 'content',
+          'weight' => '48',
+        ),
+        'nexteuropa_newsroom-newsroom_admin_tools' => array(
+          'module' => 'nexteuropa_newsroom',
+          'delta' => 'newsroom_admin_tools',
+          'region' => 'content',
+          'weight' => '49',
+        ),
+      ),
+    ),
+  );
+  $context->condition_mode = 0;
+
+  // Translatables
+  // Included for use with string extractors like potx.
+  t('newsroom');
+
+  $export['newsroom_item'] = $context;
+
+  $context = new stdClass();
+  $context->disabled = FALSE; /* Edit this to true to make a default context disabled initially */
+  $context->api_version = 3;
+  $context->name = 'newsroom_site_wide';
+  $context->description = '';
+  $context->tag = 'newsroom';
+  $context->conditions = array(
+    'context' => array(
+      'values' => array(
+        '~homepage' => '~homepage',
+        'site_wide' => 'site_wide',
+      ),
+    ),
+  );
+  $context->reactions = array(
+    'block' => array(
+      'blocks' => array(
+        'newsroom-newsroom_newsletter_subscription' => array(
```

Comment:
sometimes sidebar_right is just not available on the subsites, it shouldn't happen to see it enabled because of newsroom decided to put a newsletter subscription block there.